ui/src/widgets/ui-slider/UISlider.vue

```diff
@@ -47,6 +48,12 @@ export default {
         direction: function () {
             return this.props.height > this.props.width ? 'vertical' : 'horizontal'
         },
+        tickScaleX: function () {
+            return this.props.height > this.props.width ? 3 : 0.5
```

Comment:
3 is quit high a value, especially if lots of ticks? 2 maybe better?

Reply:
That goes into the territory of art. I'm not good at art at all. So I happily leave this kind of twaks for experts to find out.